utils/getStrapiData.js

```diff
@@ -11,7 +11,8 @@ export default async (endpoint, params) => {
       populate: 'deep',
     },
   })
-  const meta = data[0].meta
+  const components = data.components || data[0].components
+  const meta = data.meta || data[0].meta
```

Comment:
Why it has inconsistent data?
sometime string sometime array?

Reply:
So for single pages like Home page we have only one record in the table.. so strapi sends it as object.. But for pages like features and blogs we have multiple records in the table.. and we do filter to get the record which matches the slug. SO in this case strapi returns an array of objects and since the slug is unique for all the records it will only give one object inside the array.